bcManager/bcManager.py

```diff
@@ -217,16 +435,117 @@ async def bcgroup(self, ctx):
+        
+        accounts_embed.set_footer(icon_url=ctx.guild.icon_url, text="RSC Tracker Links: https://tinyurl.com/TrackerLinks")
+
+        msg : discord.Message = await ctx.send(embed=accounts_embed)
+        linked_accounts = []
+        for acc in await self.get_player_accounts(player):
+            platform = acc.get("platform").lower()
+            plat_id = acc.get("platform_id")
+            plat_name = acc.get("name")
+
+            if plat_id:
+                latest_replay = await asyncio.to_thread(self.get_latest_account_replay_by_plat_id, ctx.guild, platform, plat_id)
+                if latest_replay:
+                    player_data = await asyncio.to_thread(self.get_player_data_from_replay_by_plat_id, latest_replay, platform, plat_id)
+                    plat_name = player_data.get('name', plat_id)
+            elif plat_name:
+                player_data = await asyncio.to_thread(self.get_latest_player_data_by_platform_name, ctx.guild, platform, plat_name)
+                plat_id = player_data.get('id', {}).get('id')
+            
+           
+            if plat_id and plat_name:
+                linked_accounts.append(f"[{platform} | {plat_name}](https://ballchasing.com/player/{platform}/{plat_id})")
+            elif plat_name:
+                linked_accounts.append(f"{platform} | {plat_name}")
+
+        all_accounts_linked = " - " + "\n - ".join(linked_accounts)
+        accounts_embed.description = all_accounts_linked if linked_accounts else "No accounts have been registered."
+        await msg.edit(embed=accounts_embed)
+
 
 # endregion
 
-# region helper commands
+# region helper functions
+
+    # region listeners
+    @commands.guild_only()
+    @commands.Cog.listener("on_reaction_add")
+    async def on_reaction_add(self, reaction: discord.Reaction, user: discord.User):
+        if user.id == self.bot.user.id:
+            return
+        await self.process_ff_reacts(reaction, user, True)
+        
+    @commands.guild_only()
+    @commands.Cog.listener("on_reaction_remove")
+    async def on_reaction_remove(self, reaction: discord.Reaction, user: discord.User):
+        if user.id == self.bot.user.id:
+            return
+        await self.process_ff_reacts(reaction, user, False)
+    
+    # Listener helpers
+    def reaction_guild(self, reaction: discord.Reaction):
+        try:
+            return reaction.message.guild
+        except:
+            return None
+
+    def get_channel(self, message: discord.Message):
+        return message.channel
+
+    async def process_ff_reacts(self, reaction: discord.Reaction, user: discord.User, added: bool):
+        try:
+            message = reaction.message
+            member = message.author
+            guild = message.guild
+            channel = message.channel
+            ff_processing_data = self.ffp[guild][message]
+            if user.id is not ff_processing_data['reporter'].id:
+                await channel.send("WHAT")
```

Comment:
what's this lol

Reply:
LOL I was troubleshooting some logic. Forgot to remove it. I'll get that taken care of.